applications/IgaApplication/custom_conditions/sbm_neumann_laplacian_condition.h

```diff
@@ -0,0 +1,271 @@
+//  KRATOS  _____________
+//         /  _/ ____/   |
+//         / // / __/ /| |
+//       _/ // /_/ / ___ |
+//      /___/\____/_/  |_| Application
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+
+#pragma once
+
+// System includes
+#include "includes/define.h"
+#include "includes/condition.h"
+
+// External includes
+
+// Project includes
+#include "iga_application_variables.h"
+
+namespace Kratos
+{
+/// Condition for penalty support condition
+class KRATOS_API(IGA_APPLICATION) SbmNeumannLaplacianCondition
+    : public Condition
+{
+public:
+    ///@name Type Definitions
+    ///@{
+
+    /// Counted pointer definition of SbmNeumannLaplacianCondition
+    KRATOS_CLASS_INTRUSIVE_POINTER_DEFINITION(SbmNeumannLaplacianCondition);
+
+    /// Size types
+    using SizeType = std::size_t;
+    using IndexType = std::size_t;
+
+    ///@}
+    ///@name Life Cycle
+    ///@{
+
+    /// Constructor with Id and geometry
+    SbmNeumannLaplacianCondition(
+        IndexType NewId,
+        GeometryType::Pointer pGeometry)
+        : Condition(NewId, pGeometry)
+    {};
+
+    /// Constructor with Id, geometry and property
+    SbmNeumannLaplacianCondition(
+        IndexType NewId,
+        GeometryType::Pointer pGeometry,
+        PropertiesType::Pointer pProperties)
+        : Condition(NewId, pGeometry, pProperties)
+    {};
+
+    /// Default constructor
+    SbmNeumannLaplacianCondition() : Condition()
+    {};
+
+    /// Destructor
+    virtual ~SbmNeumannLaplacianCondition() override
+    {};
+
+    ///@}
+    ///@name Life Cycle
+    ///@{
+
+    /// Create with Id, pointer to geometry and pointer to property
+    Condition::Pointer Create(
+        IndexType NewId,
+        GeometryType::Pointer pGeom,
+        PropertiesType::Pointer pProperties
+    ) const override
+    {
+        return Kratos::make_intrusive<SbmNeumannLaplacianCondition>(
+            NewId, pGeom, pProperties);
+    };
+
+    /// Create with Id, pointer to geometry and pointer to property
+    Condition::Pointer Create(
+        IndexType NewId,
+        NodesArrayType const& ThisNodes,
+        PropertiesType::Pointer pProperties
+    ) const override
+    {

```

Comment:
```suggestion
        PropertiesType::Pointer pProperties) const override
    {
```
(just to be consistent with the rest of the file)